rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.desired_nodes/10_basic.yml

```diff
@@ -774,3 +767,135 @@ teardown:
             - { settings: { "node.external_id": "instance-000187"}, processors: 64, memory: "1b", storage: "1b", node_version: null }
   - match: { status: 400 }
   - match: { error.type: x_content_parse_exception }
+---
+"Test update desired nodes using legacy processors field":
```

Comment:
I would not call this legacy, the min/max is not really the desired way to use this.